### PR TITLE
Add resource access control config for configs/homeRealmIdentifiers endpoint

### PR DESCRIPTION
**Pull request by DinikaSen**
### Proposed changes in this pull request

This PR adds the below resource access control configuration to identity.xml.j2.

``` 
<Resource context="(.*)/api/server/v1/configs/homeRealmIdentifiers" secured="true" http-method="GET">
    <Permissions>none</Permissions>
    <Scopes>internal_login</Scopes>
</Resource>
```
Related PRs : https://github.com/wso2/identity-api-server/pull/260, https://github.com/wso2/identity-apps/pull/1581
Resolves : https://github.com/wso2-enterprise/asgardeo-product/issues/1189

### CODE REVIEW

**Review by ayshsandu — APPROVED** [review]

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml [hunk 1]
```diff
@@ -1660,6 +1660,10 @@
             <Permissions>/permission/admin/manage/identity/cors/origins/view</Permissions>
             <Scopes>internal_cors_origins_view</Scopes>
         </Resource>
+        <Resource context="(.*)/api/server/v1/configs/homeRealmIdentifiers" secured="true" http-method="GET">
```
Comment (IsuraD) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml:1663]: <Resource context="(.*)/api/server/v1/configs/home-realm-identifiers" secured="true" http-method="GET">

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 2]
```diff
@@ -2205,6 +2205,10 @@
             <Permissions>/permission/admin/manage/identity/cors/origins/view</Permissions>
             <Scopes>internal_cors_origins_view</Scopes>
         </Resource>
+        <Resource context="(.*)/api/server/v1/configs/homeRealmIdentifiers" secured="true" http-method="GET">
```
Comment (IsuraD) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2208]: <Resource context="(.*)/api/server/v1/configs/home-realm-identifiers" secured="true" http-method="GET">

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml (hunk 1 above)
Comment (DinikaSen) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml]: Renamed with https://github.com/wso2/carbon-identity-framework/pull/3349/commits/886b061624092d9b66a5fc7b95764e0b5560cab6

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 (hunk 2 above)
Comment (DinikaSen) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Renamed with https://github.com/wso2/carbon-identity-framework/pull/3349/commits/886b061624092d9b66a5fc7b95764e0b5560cab6

**Review by IsuraD — APPROVED** [review]

**Pull request merged**